spk/tt-rss/Makefile

```diff
@@ -1,43 +1,47 @@
 SPK_NAME = tt-rss
-SPK_VERS = 20221230
-SPK_REV = 16
+SPK_VERS = 20230828
+SPK_REV = 17
 SPK_ICON = src/tt-rss.png
 DSM_UI_DIR = app
 
 DEPENDS  = cross/$(SPK_NAME)
+SPK_DEPENDS = "WebStation:PHP7.4:MariaDB10:Apache2.4"
+
+# Due to not obvious WebStation handling requirements
+REQUIRED_MIN_DSM = 6.0
 
 MAINTAINER = moneytoo
 DESCRIPTION = Tiny Tiny RSS is an open source web-based news feed \(RSS/Atom\) reader and aggregator, designed to allow you to read news from any location, while feeling as close to a real desktop application as possible.
 DESCRIPTION_FRE = Tiny Tiny RSS est un agrégateur et lecteur web de flux RSS et Atom open source conçu pour vous permettre de lire des nouvelles de n\'importe quel endroit, tout en se sentant aussi proche d\'une application de bureau que possible.
-ADMIN_URL = /tt-rss/
+CHANGELOG = "Update Tiny Tiny RSS to Git revision afd04d141c (20230828)"
 DISPLAY_NAME = Tiny Tiny RSS
-CHANGELOG = "1. Update Tiny Tiny RSS to Git revision c30b24d09f (20221230) <br /> 2. Fixed broken DSM 7 initial installation procedure. <br /> 3. Upgrade PHP version to 7.4"
 
 HOMEPAGE   = https://tt-rss.org/
 LICENSE    = GPL
 
-WIZARDS_DIR = src/wizard/
-CONF_DIR = src/conf/
-
+STARTABLE = yes
+SERVICE_USER = auto
 SERVICE_SETUP = src/service-setup.sh
-SYSTEM_GROUP	 = http
 
-INSTALL_DEP_SERVICES = apache-web mysql
-START_DEP_SERVICES = apache-web mysql
+SYSTEM_GROUP = http
 
-SERVICE_COMMAND = $(INSTALL_PREFIX)/bin/tt-rss-daemon
+ADMIN_URL = /tt-rss/
 
-POST_STRIP_TARGET = tt-rss_extra_install
+WIZARDS_TEMPLATES_DIR = src/wizard_templates
+CONF_DIR = src/conf/
 
-PHP_EXTENSIONS = curl gd intl mysqli pdo_mysql posix
+include ../../mk/spksrc.common.mk
 
-override ARCH=
+# Alternate conf dir for DSM 6
+ifeq ($(call version_lt, ${TCVERSION}, 7.0),1)
+CONF_DIR = src/conf_6/
+endif
 
-# Due to not obvious WebStation handling requirements
-REQUIRED_MIN_DSM = 6.0
-INSTUNINST_RESTART_SERVICES = apache-web
+POST_STRIP_TARGET = tt-rss_extra_install
 
-SPK_DEPENDS = "WebStation:PHP7.4:MariaDB10:Apache2.4"
+PHP_EXTENSIONS := $(shell jq -r '.webservice.services[0].php.extensions[]' src/conf/resource)
```

Comment:
I don't believe you are still using this are you ?
(the point of this was to not have to redefine this list of PHP extensions)

Reply:
Agreed, removed...